src/quo/components/wallet/summary_info/view.cljs

```diff
@@ -26,34 +27,47 @@
 
 (defn networks
   [values theme]
-  (let [{:keys [ethereum optimism arbitrum]} values]
+  (let [{:keys [ethereum optimism arbitrum]} values
+        show-optimism?                       (pos? optimism)
+        show-arbitrum?                       (pos? arbitrum)]
     [rn/view
      {:style               style/networks-container
       :accessibility-label :networks}
-     [network-amount
-      {:network  :ethereum
-       :amount   (str ethereum " ETH")
-       :divider? true
-       :theme    theme}]
-     [network-amount
-      {:network  :optimism
-       :amount   (str optimism " ETH")
-       :divider? true
-       :theme    theme}]
-     [network-amount
-      {:network :arbitrum
-       :amount  (str arbitrum " ETH")
-       :theme   theme}]]))
+     (when (pos? ethereum)
+       [network-amount
+        {:network  :ethereum
+         :amount   (str ethereum " ETH")
+         :divider? (or show-arbitrum? show-optimism?)
+         :theme    theme}])
+     (when show-optimism?
```

Comment:
the designs only show these if they have a value. We could also look to remove network specific details out of this component and make it more generic but it's for another pr as this work is already growing too large.